src/azul/indexer/aggregate.py

```diff
@@ -193,8 +198,9 @@ def accumulate(self, value):
         >>> acc.get(), acc.dropped
         ([1, 2, 3], 1)
         """
+        assert not isinstance(value, set), value
```

Comment:
This is helpful as an experiment, but we don't want these in there forever. A drop! commit is the right vehicle.